generate.pl

```diff
@@ -330,15 +330,15 @@ =head1 DESCRIPTION
     && archFlag="$([ "$archBits" = '64' ] && echo '-Duse64bitall' || echo '-Duse64bitint')" \
     && ./Configure -Darchname="$gnuArch" "$archFlag" {{args}} {{extra_flags}} -des \
     && make -j$(nproc) \
-    && {{test}} \
+    && { [ "$(dpkg --print-architecture)" = 'riscv64' ] || {{test}}; } \
     && make install \
     && cd /usr/src \
     && curl -fLO {{cpanm_dist_url}} \
     && echo '{{cpanm_dist_sha256}} *{{cpanm_dist_name}}.tar.gz' | sha256sum --strict --check - \
     && tar -xzf {{cpanm_dist_name}}.tar.gz && cd {{cpanm_dist_name}} \
     && {{cpanm_dist_patch_https}} \
     && {{cpanm_dist_patch_nolwp}} \
-    && perl bin/cpanm . && cd /root \
+    && perl bin/cpanm --notest . && cd /root \
```

Comment:
Perhaps we can take this as an `ARG` with a default `--notest` so we can pass `` later in GHA builds here and below at lines 344 & 347?  That way official-image builds can proceed faster while we can affirm tests on our own CI here.